src/App.css

```diff
@@ -87,3 +87,11 @@ BEM conventions we are using:
   margin: 0;
   list-style: none;
 }
+
+.display-block {
+  display: block;
+}
+
+.display-none {
+  display: none;
+}
```

Comment:
+1 on meaningful naming.

Reply:
This is a great suggestion. We've made [a new commit](https://github.com/the-collab-lab/tcl-26-smart-shopping-list/pull/44/commits/75679e4912f07995693228cf319afa36d79a9e06) to rename these CSS classes to `.dialog-open` and `.dialog-close` so that they're related to the dialog and we can keep adding CSS to them. We've moved them into Modal.js. For now, we didn't add just a `.dialog` class but we certainly can in the future